exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = roomMaxCapacity === studentCount + mentorCount; // finish this statement
```

Comment:
This is good and works, but to help readability it's good to put calculations that go together in brackets like so:
```
let enoughSpaceInRoom = roomMaxCapacity === (studentCount + mentorCount); 
```